providers/monitor.rb

```diff
@@ -43,6 +43,7 @@ def whyrun_supported?
   service 'datadog-agent' do
     service_name node['datadog']['agent_name']
     provider service_provider unless service_provider.nil?
+    restart_command "powershell restart-service #{node['datadog']['agent_name']} -Force" if node['os'] == 'windows'
```

Comment:
for consistency with the rest of the cookbook, could you use `node['platform_family'] == 'windows'` as the condition here?

Reply:
Updated in amended commit.